docs/source/benchmarking.rst

```diff
@@ -42,8 +52,15 @@ Benchmark all of the simulated communities and their reference assignments
             --output-long ${community_size}.clustering_benchmarks.long.tsv.gz
     done
 
-Aggregate across simulated communities (when dataset index is unique)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Aggregate resulst across simulated communities
```

Comment:
This seems less descriptive to me than the previous header. Each of these header sections will be indexed along the side-bar in readthedocs. In this case, I think more descriptive section headers would be easier to quickly look-up

Reply:
Completely agreed that the headers should be descriptive. However, in my opinion, they should also be a bit concise. Headers are generally short and sweet, gives the user a gist of what's in that section.